### PR TITLE
To Do list: interactive list

### PR DESCRIPTION
**Pull request by Kayonga99**
## The following were implemented:

- No linter errors.
- Use correct git-flow.
- Document README.md file.
- Add event listener to the checkbox.

### CODE REVIEW

File: src/modules/refresh.js [hunk 1]
```diff
@@ -1,6 +1,4 @@
-/*eslint-disable*/
 import addElem from './addElement';
-import Task from './classTask';
```
Comment (thecodechaser) [inline src/modules/refresh.js:2]: - [x] "Clear all completed"  functionality is not working, When I click on the button I'm getting the following error. Please fix this issue and make sure "Clear all completed" button works as expected.

### See the image of the error

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Kayonga99👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/refresh.js [hunk 2]
```diff
@@ -30,6 +28,7 @@ const refreshList = (_class, _container) => {
         listIconDots.classList.add('hide');
         removeBtn.classList.remove('hide');
         todoList.style.backgroundColor = '#fffdcc';
+        listIconTrash.style.backgroundColor = '#F50000';
```
Comment (dasileker) [inline src/modules/refresh.js:31]: - [x] hello and nice job on adding the requested change by the previous CR, but the complete status doesn't change after selecting it as the `if (e.completed === true) {
      isChecked = 'checked';
      strikeThrough = 'strike-through';
    }` it stays false all the time


Capture web_9-7-2022_171827_gitlab com

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @Kayonga99,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## **Highlights**
- Linters are passing✅.
- Commits are consistent ✅.
- PR is well detailed ✅.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/refresh.js (hunk 2 above)
Comment (Kayonga99) [reply src/modules/refresh.js:31]: I do not understand what you are saying

**Review by Kingstalux — APPROVED** [review]
Hi @Kayonga99,
# Status: Approved ✔️  ✔️ ✔️ 

download

Good job implementing the required changes 👍 Your app is functioning as expected 🔥 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**